web/src/version.ts

```diff
@@ -0,0 +1 @@
+export const THORVG_VERSION = '0.11.5';
```

Comment:
An automatic update of the version info is necessary... Probably meson could generate this one when we build the wasm.